src/test/scala-2.11/markets/participants/OrderIssuerSpec.scala

```diff
@@ -47,12 +47,12 @@ class LiquiditySupplierSpec extends TestKit(ActorSystem("LiquiditySupplierSpec")
     val initialTick = Tick(1, 1, 1, 1, System.currentTimeMillis())
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [It's preferred to use immutable collections instead of mutable ones](https://www.codacy.com/app/davidrpugh/markets-sandbox/file/2363092232/issues/source?bid=2921361&fileBranchId=3219434#l48)